forms-flow-web/src/components/Modeller/ModelEditorHook.js

```diff
@@ -146,7 +146,8 @@ const EditModel = React.memo(
       // Deployment Source
       form.append('deployment-source', 'Camunda Modeler');
       // Tenant ID
-      if (tenant) {
+      var isApplyAllTenants = document.getElementById("apply-all-tenant-checkbox").checked;
```

Comment:
use a React State to update checkbox instead (useState), also uncheck by default

Reply:
Also we need to check the Executable to be checked by default and have at least a start event by default in create New , Have created some tickets on this will share

Reply:
Done